raiden_contracts/tests/fixtures/channel.py

```diff
@@ -385,8 +385,8 @@ def get(tx_from: HexAddress, transfers: List[Tuple]) -> None:
                 {"from": tx_from},
             )
             assert (
-                secret_registry_contract.functions.getSecretRevealBlockHeight(secrethash).call()
-                == web3.eth.block_number
+                secret_registry_contract.functions.getSecretRevealBlockTime(secrethash).call()
+                == web3.eth.get_block("latest").timestamp  # type: ignore
```

Comment:
what's the typing problem here?

Reply:
Mypy says BlockData type doesn't have a `timestamp` member.

Reply:
Weird, looks like it should:
```py
class BlockData(TypedDict, total=False):
    baseFeePerGas: Wei
    ...
    timestamp: Timestamp

Reply:
It definitely should, but:

![Screenshot 2021-12-09 at 11 28 25](https://user-images.githubusercontent.com/66514/145379643-6381a5a1-6da2-4676-b4c6-68462c4a6d62.png)

Reply:
I think I also encountered this once.
Would not be surprised to find out this is just one of the myriad of mypy bugs.

Reply:
I also get this issue all the time. For some reason I also don't get this field as completion candidate while coding. Dunno whats wrong.
Do we agree we can close this conversation here?